app/src/main/java/org/mozilla/fenix/AppRequestInterceptor.kt

```diff
@@ -26,7 +26,11 @@ class AppRequestInterceptor(private val context: Context) : RequestInterceptor {
 
         adjustTrackingProtection(host, context, session)
 
-        return null
+        return if (FeatureFlags.webChannelAuthentication) {
```

Comment:
Can you add a comment, "WebChannel-driven authentication does not require a separate redirect interceptor"

Reply:
Done!